docs/src/main/sphinx/connector/hive.rst

```diff
@@ -721,6 +723,18 @@ The connector supports the following commands for use with
     to the Trino logs and query failure messages to see which files need to be
     deleted.
 
+DROP TABLE
+^^^^^^^^^^
+
+By default, ``DROP TABLE`` operations are disabled on Hive catalogs. To
```

Comment:
Should mention that "by default" here means "only when hive.security=legacy - which is the default". i.e. if you set this property with some other `hive.security` then it'll be an error and Trino would fail to start.

```suggestion
``DROP TABLE`` operations are disabled on Hive catalogs when the `legacy` [Hive connector access control](/hive/security) is used which is the default. To
```

Reply:
link to security section .. 
and also document the other allow drop and allow alter style properties

Reply:
Done